worlds/sc2/test/test_items.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+from typing import List
+
+from worlds.sc2 import items
+
+
+class TestItems(unittest.TestCase):
+    def test_grouped_upgrades_number(self) -> None:
+        bundled_items = items.upgrade_bundles.keys()
+        bundled_item_data = [items.get_full_item_list()[item_name] for item_name in bundled_items]
+        bundled_item_numbers = [item_data.number for item_data in bundled_item_data]
+
+        check_numbers = [True if number == -1 else False for number in bundled_item_numbers]
```

Comment:
Equality check already returns a boolean, so this can just be `[number == -1 for number in bundled_item_numbers]`. imo, it's even clearer to just loop over `bundled_item_data` and assert `item_data.number == -1`.